frontend/public/components/project-overview.jsx

```diff
@@ -5,8 +5,18 @@ import * as classnames from 'classnames';
 import { Link } from 'react-router-dom';
 import { ListView } from 'patternfly-react';
 
-import { ResourceLink, resourcePath } from './utils';
+import { ResourceIcon, resourceObjPath, resourcePath } from './utils';
+
+const getRevision = (obj) => {
```

Comment:
Calculate and pass in this value from overview so that it can be used in either ProjectOverview or topology view when it is implemented?

Reply:
I think it might be better to reuse the `ControllerLink` component (although I don't know the exact use case for topology).

I sort of dislike passing in a second prop that can be easily calculated from the object... We could also export `getRevision` if we need to use it elsewhere.

Reply:
Although I guess in this case it would save us the logic of checking if it's a replica set or replication controller. Hm.

Reply:
Yeah, I think that was my main reasoning. We already have other logic to normalize the data structure for an "overview item" in the Overview component, so it might make sense to include this there. But, I'm fine with leaving it and maybe revisiting later.